app/commands/feed/import_from_opml.rb

```diff
@@ -30,9 +30,20 @@ def find_or_create_group(group_name, user)
     def create_feed(parsed_feed, group, user)
       feed = user.feeds.where(**parsed_feed.slice(:name, :url))
                  .first_or_initialize
+      find_feed_name(feed, parsed_feed)
       feed.last_fetched = 1.day.ago if feed.new_record?
       feed.group_id = group.id if group
       feed.save
     end
+
+    def find_feed_name(feed, parsed_feed)
+      return unless parsed_feed[:name].nil? && feed.name.nil?
+
+      result = FeedDiscovery.call(parsed_feed[:url])
+      title = result.title if result
+      parsed_feed[:name] =
+        ContentSanitizer.call(title.presence || parsed_feed[:url])
+      feed.name = parsed_feed[:name]
```

Comment:
It doesn't look to me like we need to assign the value back to `parsed_feed`. Can we do:

```suggestion
      feed.name = ContentSanitizer.call(title.presence || parsed_feed[:url])
```

Reply:
Good catch, changed!